plugins/bulk-import/tests/bulkImport.spec.ts

```diff
@@ -0,0 +1,34 @@
+import { expect, Page, test } from '@playwright/test';
+
+test.describe('Bulk import plugin', () => {
+  let page: Page;
+
+  test.beforeAll(async ({ browser }) => {
+    const context = await browser.newContext();
+    page = await context.newPage();
+    await page.goto('http://localhost:3000/bulk-import');
```

Comment:
We've added an [environment variable](https://github.com/janus-idp/backstage-plugins/blob/main/plugins/quay/playwright.config.ts#L20) to override the base URL for the other plugins. Can you do that for the sake of consistency?